cfn-resources/data-lake-pipeline/mongodb-atlas-datalakepipeline.json

```diff
@@ -1,6 +1,6 @@
 {
   "typeName": "MongoDB::Atlas::DataLakePipeline",
-  "description": "An example resource schema demonstrating some basic constructs and validation rules.",
+  "description": "Data Lake is deprecated. As of September 2024, Data Lake is deprecated and will reach end-of-life. It will be removed on September 30, 2025. If you use Data Lake, you should migrate to alternative solutions before the service is removed. To learn more, see <https://dochub.mongodb.org/core/data-lake-deprecation>. This resource returns,, adds, edits, and removes data lake pipelines.",
```

Comment:
I don't think we should include the end date. At least we didn't do it in TF or Atlas CLI

Reply:
I copied this from the TF [PR](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2599/files) which includes the end date

Reply:
Oops. I forgot to update the PR. Created a new one: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2616

Reply:
Updated in this PR also